sklearn/ensemble/_stacking.py

```diff
@@ -592,9 +607,20 @@ def fit(self, X, y, sample_weight=None):
             Returns a fitted instance of estimator.
         """
         check_classification_targets(y)
-        self._le = LabelEncoder().fit(y)
-        self.classes_ = self._le.classes_
-        return super().fit(X, self._le.transform(y), sample_weight)
+        if type_of_target(y) == "multilabel-indicator":
```

Comment:
Is this right? This suggestion might needed black formatting.
```suggestion
        if type_of_target(y) in ("multilabel-indicator", "multiclass-multioutput"):
```

Reply:
Yes, that's true as well. I think we need to extend the test suit to account for the `multiclass-multioutput` case, even though this wasn't the initial focus of this PR.

Reply:
IMO, we can also enlarge the scope of this PR to include this case if needed.